iPetCare/Application/Services/ExaminationParameterService.cs

```diff
@@ -102,6 +102,9 @@ public async Task<ServiceResponse<UpdateExaminationParameterDtoResponse>> Update
                 return new ServiceResponse<UpdateExaminationParameterDtoResponse>(HttpStatusCode.OK, responseDto);
             }
 
+            if (result == 0)
```

Comment:
Czy jeśli nie nastąpiła żadna zmiana, tzn ktoś wysłał requesta identycznego do tego, co znajduje się w bazie to czy traktujemy to jako błąd? W sumie to summon @blc132

Reply:
Właśnie nie wiem jak tutaj dawać? Powinnam też zwracać OK w przypadku braku zmiany? No w sumie to nie jest błąd, to może OK byłoby lepsze